index.bs

```diff
@@ -696,6 +698,39 @@ authorizing an authenticator with which to complete the operation.
 </div>
 
 
+### Signal platform authenticator capability - isPlatformAuthenticatorReady() method ### {#isPlatformAuthenticatorReady}
+
+<div link-for-hint="WebAuthentication/isPlatformAuthenticatorReady">
+This method is used to signal users whether they can log in to the [RP] with the <a>platform-attached</a> authenticator. 
```

Comment:
I am a bit confused about "help them set up". AFAIK, the reason why this method is needed is because users who own a platform authenticator often don't know they can log in to a website with the authenticator. The goal of the method is to promote usage of the WebAuthn function among users who own platform authenticators, not to increase the number of users who activated the authenticator, which should be handled by the platform.